gulpfile.js

```diff
@@ -106,9 +106,11 @@ var cssFilePaths = [];
 var jsFilePaths = [];
 var fontFolderPaths = [];
 var cssBackgroundFilepaths = [];
+// This cache slug should be same as feconf.CACHE_SLUG_DEV
```

Comment:
I don't see a good reason for why we'd want to use a dev slug, so I lean towards removing it.